### PR TITLE
feat(notion): fix status, respect database names, delete database, colors

### PR DESCRIPTION
**Pull request by erics118**
## Description

This PR fixes support for the `status` property in numerous places:
- database list view
- database kanban view
- create page view

Also, before, the action panel did called "Databases" as "Pages", which has now been fixed.

Previously, databases could not be deleted because the command would try to delete it as a page, which has now been fixed.

Additionally, colors now use the colors provided by Notion, rather than the limited pre-set colors provided by Raycast.

## Screencast

<details>

<summary>Only the create page view has noticable differences.</summary>
<br/>
(The `Status` property here is of the `status` type

Before:
     
CleanShot 2023-09-03 at 11 51 55

After:

CleanShot 2023-09-03 at 11 52 39

</details>

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/notion/src/components/PageListItem.tsx [hunk 1]
```diff
@@ -163,20 +167,23 @@ export function PageListItem({
             <ActionCreateQuicklink page={page} />
 
             <Action
-              title="Delete Page"
+              title={`Delete ${capitalize(page.object)}`}
```
Comment (thomaslombart) [inline extensions/notion/src/components/PageListItem.tsx:170]: It's probably better to create a variable as it's used many times later on:

```tsx
const pageWord = capitalize(page.object)
```

**Review by thomaslombart — COMMENTED** [review]
Thanks for the fixes! 💪

**Comment by erics118** [comment]
@thomaslombart fixed, if you didn't notice

**Review by thomaslombart — APPROVED** [review]
Great additions! 👏

**Comment by thomaslombart** [comment]
@erics118 Oh, I've just seen something. Could you add an entry in the `CHANGELOG.md` file? Here's our guidelines about that: [Version History](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history).

**Comment by erics118** [comment]
Added the changelog entry.

**Comment by erics118** [comment]
fixed linked account, did the credits get added?

**Pull request merged**